### PR TITLE
Adding service for parks. Hooking up parks list page. Adding keycloak.

### PR DESCRIPTION
**Pull request by danieltruong**

### CODE REVIEW

File: src/app/parks/park-list/park-list.component.ts [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
   selector: 'app-park-list',
   templateUrl: './park-list.component.html',
   styleUrls: ['./park-list.component.scss']
 })
-export class ParkListComponent implements OnInit {
+export class ParkListComponent implements OnInit, OnDestroy {
+  private alive = true;
   // Component
   public loading = true;
   // This will be changed to service.
```
Comment (mtCarto) [inline src/app/parks/park-list/park-list.component.ts:16]: remove comment, looks like service is in

**Review by mtCarto — APPROVED** [review]

File: src/app/parks/park-list/park-list.component.ts (hunk 1 above)
Comment (danieltruong) [reply src/app/parks/park-list/park-list.component.ts]: e29d540

File: src/env.js [hunk 2]
```diff
@@ -7,13 +7,11 @@
...
-  window.__env.BANNER_COLOUR = 'red'
-  window.__env.API_LOCATION = 'http://localhost:3000';
-  window.__env.API_PATH = '/api';
-  window.__env.API_PUBLIC_PATH = '/api/public';
-  window.__env.KEYCLOAK_CLIENT_ID = 'eagle-admin-console';
-  window.__env.KEYCLOAK_URL = 'https://oidc.gov.bc.ca/auth';
-  window.__env.KEYCLOAK_REALM = 'eagle';
+  window.__env.API_LOCATION = 'https://pkqlwkdzka.execute-api.ca-central-1.amazonaws.com';
+  window.__env.API_PATH = '/dev';
+  window.__env.KEYCLOAK_CLIENT_ID = 'nrpti-admin';
```
Comment (cameronpettit) [inline src/env.js:12]: assuming this will change away from `nrpti-admin` at a later date?

**Review by cameronpettit — APPROVED** [review]

File: src/env.js (hunk 2 above)
Comment (danieltruong) [reply src/env.js:12]: That's correct. We don't have a Parks realm yet.

**Pull request merged**